store/store.go

```diff
@@ -1977,3 +1972,16 @@ func (s *Store) ReadyToBuy(user *auth.UserState) error {
 		return &errors
 	}
 }
+
+func (s *Store) CacheDownloads() int {
+	return s.cfg.CacheDownloads
+}
+
+func (s *Store) SetCacheDownloads(amount int) {
```

Comment:
Amount feels a bit ambiguous here. Amount of data or amount of files? What's the unit? nfiles or fileCount might be more clear.